### PR TITLE
New Quest - HM01 Cut (Lance + SSAnne + All Items)

### PR DESCRIPTION
**Pull request by Rympex**
Tested with secondary account and work perfectly :)

### CODE REVIEW

**Comment by ghost** [comment]
Remember to also add the quests in [https://github.com/g0ldPRO/Questing.lua/blob/master/Quests/QuestManager.lua](url) ;)

File: Quests/Kanto/SSAnneQuest.lua [hunk 1]
```diff
@@ -0,0 +1,288 @@
...
+		elseif isNpcOnCell(14, 11) and not TrashBin3Kitchen then -- Item: Hyper Potion
+			if not dialogs.TrashBinCheck.state then
+				return talkToNpcOnCell(14, 11)
+			else
+				dialogs.TrashBinCheck.state = false
+				TrashBin3Kitchen = true
+				return
+			end
+		else
+			return talkToNpcOnCell(5, 3)
```
Comment (ghost) [inline Quests/Kanto/SSAnneQuest.lua:253]: I get `error: talkToNpcOnCell: could not find any NPC on the cell [5,3].`
The NPC is in fact on that cell but it's not detected (I tested it multiple times).
I changed to `return talkToNpc("Doctor Neumann")` and it worked.

**Comment by Phibonacci** [comment]
As already discussed with Rympex, SSAnne should be a quest on its own since being on SSAnne map will always mean we are doing the SSAnne quest.

**Comment by g0ldPRO** [comment]
As already discussed with Rympex, SSAnne should be a quest on its own since being on SSAnne map will always mean we are doing the SSAnne quest.

**Comment by Silv3rPRO** [comment]
:+1: working nicely.
Talking twice to the NPC, but not a big deal.

**Comment by Rympex** [comment]
Edit done i will post the new quest

**Pull request closed without merging**